### PR TITLE
Show failed deployment names in listing table

### PR DESCRIPTION
**Pull request by AlaaElattar**
- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/1214

image

### CODE REVIEW

File: packages/playground/src/utils/load_deployment.ts [hunk 1]
```diff
@@ -27,7 +28,7 @@ export async function loadVms(grid: GridClient, options: LoadVMsOptions = {}) {
         `%c[Error] failed to load deployment with name ${name}:\n${normalizeError(e, "No errors were provided.")}`,
         "color: rgb(207, 102, 121)",
       );
-      return null;
+      failedDeployments.push(name);
```
Comment (AhmedHanafy725) [inline packages/playground/src/utils/load_deployment.ts:31]: is it the only place we list the machines, and what about the kubernetes list?

Comment (AlaaElattar) [reply packages/playground/src/utils/load_deployment.ts:31]: will add it for kubernetes also

Comment (AlaaElattar) [reply packages/playground/src/utils/load_deployment.ts:31]: image

File: packages/playground/src/components/vm_deployment_table.vue [hunk 2]
```diff
@@ -108,6 +113,8 @@ async function loadDeployments() {
   if (chunk3.count > 0) await grid!.gateway.list();
 
   const vms = mergeLoadedDeployments(chunk1, chunk2, chunk3 as any);
+  const failedDeployments = [...(chunk1 as any).failedDeployments, ...(chunk2 as any).failedDeployments];
```
Comment (AhmedHanafy725) [inline packages/playground/src/components/vm_deployment_table.vue:116]: chunk3 is missing?

**Review by maayarosama — APPROVED** [review]

**Pull request merged**